doc/guide/sanity-checklist.rst

```diff
@@ -0,0 +1,98 @@
+.. _sanity-checklist:
+
+#######################################
+Flux Accounting Health/Sanity Checklist
+#######################################
+
+********
+Overview
+********
+
+As flux-accounting is deployed and running on a system, there are a number of
+components an administrator can check to ensure that the module is loaded and
+running as expected. The following attempts to list the major tasks an
+administrator can look to complete after building, installing, and configuring
+flux-accounting on a system.
+
+*****************************
+Database Integrity and Access
+*****************************
+
+Ensure users, banks, queues, and projects can be added, viewed, and edited with
+various ``flux account`` commands:
+
+.. code-block:: console
+
+    $ flux account add-bank --parent-bank=root giants 1
+    $ flux account add-user --username=crawford --bank=giants
+    $ flux account add-queue orange
+    $ flux account add-projects shortstops
```

Comment:
Nice work! Maybe add a note that after the sanity check is complete, all of these commands work backwards with `flux account delete-*`?